tests/SimplifyTest.elm

```diff
@@ -6959,6 +6961,1488 @@ a = False
+            \() ->
+                """module A exposing (..)
+a = List.foldl (\\_ -> identity) x list
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "The call to List.foldl will result in the initial accumulator"
+                            , details = [ "You can replace this call by the initial accumulator." ]
+                            , under = "List.foldl"
+                            }
+                            |> Review.Test.whenFixed """module A exposing (..)
+a = x
+"""
+                        ]
+        , test "should replace List.foldl (\\_ a -> a) x list by x" <|
+            \() ->
+                """module A exposing (..)
+a = List.foldl (\\_ a -> a) x list
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "The call to List.foldl will result in the initial accumulator"
+                            , details = [ "You can replace this call by the initial accumulator." ]
+                            , under = "List.foldl"
+                            }
+                            |> Review.Test.whenFixed """module A exposing (..)
+a = x
+"""
+                        ]
+        , test "should replace List.foldl (always <| \\a -> a) x list by x" <|
+            \() ->
+                """module A exposing (..)
+a = List.foldl (always <| \\a -> a) x list
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "The call to List.foldl will result in the initial accumulator"
+                            , details = [ "You can replace this call by the initial accumulator." ]
+                            , under = "List.foldl"
+                            }
+                            |> Review.Test.whenFixed """module A exposing (..)
+a = x
+"""
+                        ]
+        , test "should replace List.foldl (always identity) x by always x" <|
+            \() ->
+                """module A exposing (..)
+a = List.foldl (always identity) x
+"""
+                    |> Review.Test.run (rule defaults)
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "The call to List.foldl will result in the initial accumulator"
+                            , details = [ "You can replace this call by the initial accumulator." ]
+                            , under = "List.foldl"
+                            }
+                            |> Review.Test.whenFixed """module A exposing (..)
+a = always x
+"""
+                        ]
+        , Test.concat listFoldlSumTests
```

Comment:
I didn't even know `Test.concat` existed! Though I'd say it's probably nicer to have a describe for each one (at their declaration, not here)

Reply:
Agreed. Wasn't sure since your tests for a single function were never split off into separate describes I think